react-bootstrap/index.d.ts

```diff
@@ -528,6 +528,7 @@ declare namespace ReactBootstrap {
 
     // <Navbar.Toggle />
     interface NavbarToggleProps {
+        onClick?: React.MouseEventHandler<{}>;
```

Comment:
Handlers signatures for other components don't specify element types but sure, I can make it more strict.